driver/razercommon.c

```
@@ -86,7 +86,7 @@ int razer_get_usb_response(struct usb_device *usb_dev, uint report_index, struct
 
     // Send the request to the device.
     // TODO look to see if index needs to be different for the request and the response
-    retval = razer_send_control_msg(usb_dev, request_report, report_index, wait_min, wait_max);
+    razer_send_control_msg(usb_dev, request_report, report_index, wait_min, wait_max);
```

Perhaps we should add some error handling instead... Not good to ignore errors

---

Indeed, I'll revert the change and try to add error handling in a different PR